database.sql

```diff
@@ -6,3 +6,15 @@ CREATE TABLE public.task
     due_date timestamp with time zone,
     is_done boolean
 )
+CREATE TABLE public.label
+(
+    id SERIAL,
+    name text COLLATE pg_catalog."default",
+    color text COLLSTE pg_catalog."default",
```

Comment:
Should name and color be set to unique? Or maybe just name?

Reply:
Also I looked up `COLLATE pg_catalog."default"` and it looks like it is how it handles sorting strings and I think what we have it is the default so we don't need it